src/Data/Dwarf.hs

```diff
@@ -528,21 +600,33 @@ getCUHeader (endian, dwarfSections) abbrevMapCache (CUOffset cuOff) = do
   (enc, unitLength) <- getDwarfSize endian
   postLength <- Get.bytesRead
   let totalLength = fromIntegral (postLength - start) + unitLength
-  _version <- derGetW16 endian
-  abbrevOffset <- desrGetOffset endian enc
-  addr_size <- getWord8
+  version <- derGetW16 endian
+  unitType <- if version >= 5 then Just <$> getWord8 else pure Nothing
+  -- in v5 address_size first, in v4 abbrevoffset
+  (abbrevOffset, addr_size) <- if version >= 5 then
```

Comment:
Let's keep our names consistent:
```suggestion
  (abbrevOffset, addrSize) <- if version >= 5 then
```

Reply:
While I would typically agree i am keeping the casing of addr size consistent with the current repo.